### ISSUE TITLE
meta_mpd.py': [Errno 13] Permission denied

### ISSUE THREAD

**Issue by tgurr** [badaix/snapcast#970]
**Describe the bug**
`Jan 05 16:28:07 host snapserver[633134]: Stream: steamname, message: python3: can't open file '/usr/share/snapserver/plug-ins/meta_mpd.py': [Errno 13] Permission denied`

**Steps to Reproduce**
1. Install snapserver 0.26.0
2. Configure to use controlscript
3. Try to start it
4. See error in journal 

`/usr/share/snapserver/plug-ins/meta_mpd.py` is having only executable rights but no read rights.

```
# ls -la /usr/share/snapserver/plug-ins/meta_mpd.py 
---x--x--x 1 root root 33278 Jan  5 16:12 /usr/share/snapserver/plug-ins/meta_mpd.py
```

Workaround:

```
# chmod 755 /usr/share/snapserver/plug-ins/meta_mpd.py
# ls -la /usr/share/snapserver/plug-ins/meta_mpd.py 
-rwxr-xr-x 1 root root 33278 Jan  5 16:12 /usr/share/snapserver/plug-ins/meta_mpd.py
```

**Environment details**
- OS: Exherbo Linux
- Snapcast version: 0.26.0
- Source-based distribution, compiled from package.

```log
Jan 05 16:28:07 host snapserver[633134]: Stream: streamname, message: python3: can't open file '/usr/share/snapserver/plug-ins/meta_mpd.py': [Errno 13] Permission denied
```

**Comment by badaix** [comment]
Should be fixed with 7ab3daa76d9bd2700b609162c6cbe47351e535d0

**Comment by badaix** [comment]
Fixed in [snapcast v0.27.0](https://github.com/badaix/snapcast/releases/tag/v0.27.0)
